### DISCUSSION TITLE
Internal Assert Failed Error

### DISCUSSION THREAD

**Discussion by Eddowesselink** [Project-MONAI/MONAI#4124] [General]
Dear contributors,

I'm running into a pytorch bug after training some _k_ of iterations. Will get two errors, one stating the RandomCropbyPosNegLabel failed and an Internal Assert Failed error:

'out_ptr == out_accessor[thread_count_nonzero[tid + 1]].data()INTERNAL ASSERT FAILED at ..\\aten\\src\\ATen\\native\\TensorAdvancedIndexing.cpp:1723, please report a bug to PyTorch.'

At this moment using pytorch version1.10.2+cu113

Anyone familiar with this bug / can anyone help me with this issue?

Kind regards, 
Eddo

**Comment by Nic-Ma** [comment]
Hi @Eddowesselink ,

Thanks for your interest here.
Seems like some PyTorch known issue: https://github.com/pytorch/pytorch/issues/67864?

Thanks.

**Comment by Eddowesselink** [comment]
Hi @Nic-Ma ,

Thanks for this! 

Kind regards,
Eddo

**Comment by Kin-Zhang** [comment]
Have same question on torch 2.0.1, check issue and the solution but still pop up the question, running on a big cluster...

1. https://github.com/pytorch/pytorch/issues/67864#issuecomment-972927068
2. https://github.com/pytorch/pytorch/issues/67864#issuecomment-974468364
3. https://github.com/pytorch/pytorch/issues/67864#issuecomment-1095271966

Tried what they mentioned but still randomly with this error
